images/iso/isolinux/grub-new-format.cfg

```diff
@@ -0,0 +1,53 @@
+if loadfont /boot/grub/font.pf2 ; then
+    set gfxmode=auto
+    insmod efi_gop
+    insmod efi_uga
+    insmod gfxterm
+    terminal_output gfxterm
+fi
+
+set menu_color_normal=white/black
+set menu_color_highlight=black/light-gray
+
+set timeout=30
+
+loadfont unicode
+
+menuentry "Install ONA (DHCP)" {
+    set gfxpayload=keep
+    linux   /casper/vmlinuz debug autoinstall ds=nocloud\;s=/cdrom/nocloud-dhcp/ ---
+    initrd  /casper/initrd
+}
+
+menuentry "Install ONA (Static IP)" {
```

Comment:
Can we change the order? Static IP is the preferred one.

Reply:
Switched order of menu entries.